models/index copy.js

```diff
@@ -0,0 +1,56 @@
+const Sequelize = require('sequelize')
```

Comment:
This file shouldn't have been committed. Always make sure you are keeping your github clean